spot_driver/spot_driver/spot_ros2.py

```diff
@@ -794,7 +794,7 @@ def cmdVelCallback(self, data):
         if not self.spot_wrapper:
             self.node.get_logger().info('Mock mode, received command vel ' + str(data))
             return
-        self.spot_wrapper.velocity_cmd(data.linear.x, data.linear.y, data.angular.z)
+        self.spot_wrapper.velocity_cmd(data.linear.x, data.linear.y, data.angular.z, self.cmd_duration)
```

Comment:
Should there be a corresponding submodule update to `spot_wrapper` to support this environment?